tests/validate/common/includes/cmdparser/cmdlineparser.h

```diff
@@ -88,7 +69,7 @@ class CmdLineParser {
   /*!
    * retrieve value using a key
    */
-  string value(const char *key);
+  std::string value(const char *key);
```

Comment:
Looking at the data members in this class they are all std::string, so presumably 'key' will converted to a std::string when looking up in one of the maps,  The conversion should be done at the API level so the signature should be `std::string value(const std::string& key)`.  This makes the API more flexible as it can be called with both a char* and a std::string.